Sources/String+Sweetness.swift

```diff
@@ -1,5 +1,14 @@
+import Foundation
+
 public extension String {
     public var length: Int {
         return self.characters.count
     }
+
+    public var isValidEmail: Bool {
+        let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"
```

Comment:
Yup! I'll add the method to SweetFoundation instead. If you have any recommendation for a better regEx feel free to suggest!